content/snowplow-recommendations/model/run.md

```diff
@@ -0,0 +1,66 @@
+    
+3. You will also need to add the model name(s) to the list parameter in the config file (i.e. `tables_to_export`).
+4. Create a schema for each model in the `aws_personalize_utilities/schemas/` folder. They must be named `media_items.avro` and `media_users.avro` for the python script that creates the recommenders to read them. You can copy the formatting of the ecommerce schemas or see the documentation [here](https://docs.aws.amazon.com/personalize/latest/dg/VIDEO-ON-DEMAND-datasets-and-schemas.html).
+{{% /expand %}}
+
+#### **Step 1:** Update the vars in your `dbt_project.yml` file:
+
+The current version of this package will only build the models (datasets) for the E-commerce domain (i.e. 'interactions', 'items' and 'users') whereas for the Video On Demand domain only the 'interactions' model will be created. This is due to the fact that the current media tracking available doesn't capture some data that are required fields for the 'items' and 'users' datasets for VOD. You are able to create them yourself if you have the available fields in other tables.
+   
+You may wish to only run the models for a specific domain (or both) by enabling them in the project config file via:
+```yaml
+models:
+  snowplow_recommendations:
+    +materialized: table
+    ecommerce:
+      enabled: true
+    media:
+      enabled: true
+```
+
+You may need to further update some variables in your `dbt_project.yml` file:
+- `tables_to_export`: When a domain type is enabled in the config file then all the models for that domain will be built, but you may list which tables (models) to be exported to S3 with this variable here. Enter the name of the model in this array if you'd like a table to be exported to S3 to be trained by AWS Personalize (note that the 'interaction' models are required datasets but the 'items' and 'users' datasets are optional).
+- `snowflake_stage_name`: This will be the name of the external Stage in your Snowflake account to load data to AWS S3. If you used Terraform to create this stage then the name can be found in the TF variable files.
+- `external_catalog`: This is the name of the external catalog on your Databricks account. In order to create external tables the query must be executed from an external catalog. If you used Terraform to create this stage then the name can be found in the TF variable files.
+- `s3_bucket`: This is the S3 path (i.e. `s3://<bucket name>/`) for which your Databricks' external location is configured to. If you used Terraform to create this stage then the name can be found in the TF variable files.
+
+The other variables behave similarly to all the other Snowplow dbt packages and their configurable variables for running the models.
+
+Run `dbt deps` followed by `dbt run` in the `dbt-snowplow-recommendations` directory. This will create the table(s) in your data warehouse and export the models as csv files to the S3 bucket.
```

Comment:
Given we are expecting the user to import this package, it is not obvious what all the variables we have available are (especially as we don't currently have this in the docs). I think it would be best to list all variable, required and optional ones in a table.